<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cadastro</class>
 <widget class="QDialog" name="Cadastro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Cadastro{background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(24, 22, 83, 255), stop:0.505682 rgba(94, 90, 221, 255), stop:1 rgba(24, 22, 83, 255))}

QLabel{color:&quot;white&quot;;}

QMessageBox QLabel {
    color:&quot;black&quot;;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Digite o nome e o CPF:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Digite o RG:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FilaCirc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cadastrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SairCadastro">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>571</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Cadastrar nome e CPF</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Ler o Arquivo &quot;Cadastrados&quot;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>571</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSpinBox" name="RG">
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="resultado">
      <property name="text">
       <string>Confirmação do RG.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>SairCadastro</sender>
   <signal>clicked()</signal>
   <receiver>Cadastro</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>512</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
